app/src/main/java/com/example/composetemplate/presentation/common/LoginTextField.kt

```diff
@@ -36,9 +36,11 @@ fun LoginTextField(
     isValid: Boolean,
     onValueChange: (String) -> Unit,
     isLastEditText: Boolean = false,
+    isFirstEditText: Boolean = false,
```

Comment:
Maybe is better to pass index instead of boolean isLastEditText &  isFirstEditText, maybe tomorrow we will be care about the second or the third.